pkg/scheduler/core/equivalence/eqivalence.go

```diff
@@ -68,12 +68,16 @@ func NewCache() *Cache {
 type NodeCache struct {
 	mu    sync.RWMutex
 	cache predicateMap
+	// Flags which indicate if we received invalidation requests for each
+	// predicate during check was in flight.
+	receivedInvalidatioRequests map[string]bool
```

Comment:
Rename to `receivedInvalidationRequests`

Reply:
Fixed.